demos/chat-demo-mpp/gradle/libs.versions.toml

```diff
@@ -1,16 +1,16 @@
 [versions]
-kotlin = "1.8.20"
+kotlin = "1.9.10"
 agp = "8.0.2"
-compose = "1.4.0"
+compose = "1.5.3"
 supabase = "1.1.0"
-koin = "3.4.1"
-ktor = "2.3.1"
+koin = "3.4.3"
+ktor = "2.3.7"
 
 androidx-core = "1.10.1"
 androidx-compat = "1.6.1"
 androidx-lifecycle = "2.6.1"
 androidx-activity = "1.7.2"
-
+org-jetbrains-kotlin-jvm = "1.9.10"
```

Comment:
Two things:
- We can use the newest possible versions if we are updating them anyways (Kotlin 1.9.21, Compose 1.5.11)
- Is there a reason kotlin-jvm has a separate version entry?

Reply:
- Let me try the latest posible versions
- No it's not. It should be the same version with Kotlin. I'll update it

Reply:
Just updated. All good now